### ISSUE TITLE
nsWebViewBridge undefined in {NS}-6.3.0 with Vue

### ISSUE THREAD

**Issue by Rho-bur** [Notalib/nativescript-webview-ext#58]
### Same issue as the #48 except that I don't even get the nsWebViewBridge in the window object, everything else loads correctly. 

@m-abs  Webpack.config.js does contain { from: { glob: "assets/**/*" } } all my assets being in subdirs.

### Which platform(s) does your issue occur on?
- Android 6.3.1
- Android 9
- emulator Pixel_9.0 built with Android Studio on MacOs Catalina.

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.3.0
- iOS: XCode 11.3.1.
- nodejs: v11.15.0
- tns-core-modules: tns-core-modules@6.3.2 
- Runtime(s): tns-android 6.3.1 tns-ios 6.3.0
- Plugin(s): Tried with @nota/nativescript-webview-ext@6.2.1 and @nota/nativescript-webview-ext@6.3.0-alpha.1

### Please, tell us how to recreate the issue in as much detail as possible.
Describe the steps to reproduce it.

### Is there any code involved?

Vue:
```
<template>
    <StackLayout>
              <WebViewExt
                      autoInjectJSBridge="true"
                      supportXLocalScheme="true"
                      src="~/assets/static/map.html"
                      builtInZoomControls="false"
                      cacheMode="default"
                      databaseStorage="false"
                      debugMode="true"
                      displayZoomControls="false"
                      domStorage="true"
                      viewPortSize="initial-scale=1.2"
                      @loadFinished="logMe"
                      width="100%"
                      height="100%"
              />

<template>
    </StackLayout>
methods : {
   logMe(args) {
                args.object.emitToWebView("hasMarkers", this.getArea);
            }
}

```
map.html:

```
window.addEventListener('DOMContentLoaded', (event) => {
window.nsWebViewBridge.on('hasMarkers', (data) => {
                console.log('markers event from native');
                console.dir(data);
        });
    });
```

**Comment by Rho-bur** [comment]
Hi guys,
I know this is a pro-bono work you offer to community and I am grateful for your doing it.
I guess your time is limited but I'd appreciate if you at least could confirm that this is a known issue.

**Comment by m-abs** [comment]
Hi @Rho-bur 

It is an annoying timing issue. `nsWebViewBridge` is ready just after `load`.

Try this.

```javascript
window.addEventListener('load', (event) => {
setTimeout(() => {
    window.nsWebViewBridge.on('hasMarkers', (data) => {
                console.log('markers event from native');
                console.dir(data);
        });
}, 10);
    });
``` 

I'm making a custom event to make it more clear when it is ready.

One minor thing `supportXLocalScheme` is for internal use.

**Comment by m-abs** [comment]
I just publish `v6.3.0-alpha.3` could you try this for me?

```javascript
window.addEventListener('ns-brige-ready', (event) => {
    const nsWebViewBridge = event.detail || window.nsWebViewBridge;

    nsWebViewBridge.on('hasMarkers', (data) => {
                console.log('markers event from native');
                console.dir(data);
    });
 });
```

**Comment by Rho-bur** [comment]
Hi @m-abs ,
Thank you for taking the time to help on this issue!
I have tried with alpha.3 as you recommended above and I am happy to confirm it works, I get my data on the webview :-).
I'll close this issue immediately after you read the next lines (as I am not sure that if closing it now will get you notified).
A curiosity, not important now that you came up with the event fix but I am a bit puzzled.
Before your answer I tried several things in my "desperation" to have this working; I discovered that using the  executeJavaScript function you provide I was able to send my data to webview.
Ex.
Native:
```
args.object.executeJavaScript("callFromNativeScript(" + markersSent + ")");
```
Webview:
```
    function callFromNativeScript(msg) {
        let msgString = JSON.stringify(msg);
        console.log('markers count on webview typeof ' + typeof msg + ' length ' + msgString.length);
        const markersReceived = JSON.parse(msgString);
        console.log('markersReceived ' + msgString);
}
```
It works and can pass the data to webview anytime however it appears that before reaching the webview it somehow convert the stringified sent data.
Without using the  JSON.stringify(msg) on the receiving end the data length differs a bit, with using  JSON.stringify(msg) the data length is the same although it require some hydrating.
Didn't have the time to look into the executeJavaScript function itseld because you obliged with the fix :-).
Is this "feature" intended? One can use it as an alternative to events?
